### PR TITLE
leg 280 Wallet balance styling

### PR DESCRIPTION
**Pull request by Th3Doo**
# Description
- Wallet balance fonts
- Wallet balance button styling

### CODE REVIEW

File: src/pages/Wallet.vue [hunk 1]
```diff
@@ -83,44 +83,61 @@ export default defineComponent({
...
-          | My Balance
+        .text-wallet-heading.text-grey-8
+          | Balance
       q-card-section
-        .text-bold 
-          | {{ balance }} LEGAL
-        .text
-          | {{ gbpValue }} (GBP)
+        .text-wallet-topline.text-bold 
+          | {{ balance }} LCP (LEGALCOIN POUND)
```
Comment (Dewald928) [inline src/pages/Wallet.vue:92]: Should be LEGAL not lcp. Change everywhere.

**Review by Dewald928 — COMMENTED** [review]
Looks good. Everything is LEGAL tokens and GBP

File: src/pages/Wallet.vue (hunk 1 above)
Comment (Th3Doo) [reply src/pages/Wallet.vue:92]: Done

**Review by Dewald928 — APPROVED** [review]
Happy. Merging

**Pull request merged**